CHANGELOG.md

```diff
@@ -6,6 +6,15 @@
 
 * [#8912](https://github.com/rubocop-hq/rubocop/pull/8912): Fix `Layout/ElseAlignment` for `rescue/else/ensure` inside `do/end` blocks with assignment. ([@miry][])
 * [#8971](https://github.com/rubocop-hq/rubocop/issues/8971): Fix a false alarm for `# rubocop:disable Lint/EmptyBlock` inline comment with `Lint/RedundantCopDisableDirective`. ([@koic][])
+### New features
```

Comment:
The changelog seems messed up. I think you didn't rebase properly.

Reply:
Yes, yes it does. Sorry, will look at that right meow.

Reply:
Alright, that looks a little better.